admin/src/pages/CommunityStatistic.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <div class="community-statistic">
+    <statistic-table v-model="statistics" />
+  </div>
+</template>
+<script>
+import { communityStatistics } from '@/graphql/communityStatistics.js'
+import StatisticTable from '../components/Tables/StatisticTable.vue'
+
+export default {
+  name: 'CommunityStatistic',
+  components: {
+    StatisticTable,
+  },
+  data() {
+    return {
+      statistics: {
+        totalUsers: null,
+        activeUsers: null,
+        deletedUsers: null,
+        totalGradidoCreated: null,
+        totalGradidoDecayed: null,
+        totalGradidoAvailable: null,
+        totalGradidoUnbookedDecayed: null,
+      },
+    }
+  },
+  methods: {
+    getCommunityStatistics() {
+      this.$apollo
+        .query({
+          query: communityStatistics,
+        })
+        .then((result) => {
+          this.statistics.totalUsers = result.data.communityStatistics.totalUsers
+          this.statistics.activeUsers = result.data.communityStatistics.activeUsers
+          this.statistics.deletedUsers = result.data.communityStatistics.deletedUsers
+          this.statistics.totalGradidoCreated = result.data.communityStatistics.totalGradidoCreated
+          this.statistics.totalGradidoDecayed = result.data.communityStatistics.totalGradidoDecayed
+          this.statistics.totalGradidoAvailable =
+            result.data.communityStatistics.totalGradidoAvailable
+          this.statistics.totalGradidoUnbookedDecayed =
+            result.data.communityStatistics.totalGradidoUnbookedDecayed
+        })
+        .catch(() => {
+          this.toastError('communityStatistics has no result, use default data')
+        })
+    },
+  },
+  created() {
+    this.getCommunityStatistics()
```

Comment:
use the apollo object in the instead of this. See new example: https://github.com/gradido/gradido/blob/master/admin/src/pages/CreationConfirm.vue#L116

Tests might have to be adjusted accordingly - how to test is available in the `CreationConfirm` Test file.

Reply:
Nice.
This has been changed by https://github.com/gradido/gradido/commit/fa78db6d8b6084d518277708afbd1ba2aa6b22d3.